src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  background: #fff;
+  box-shadow: 0.01px 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-left: 50px;
```

Comment:
instead of adding margin-left to just logo you can add padding to the whole header

Reply:
also with `align-items: center;` on header you no longer need margin-top on logo, it will be centered by flexbox